bin/i18n/resources/pegasus/hourofcode.rb

```diff
@@ -0,0 +1,50 @@
+require 'fileutils'
+require 'json'
```

Comment:
if we did not require json before why do we require it now? we are still dealing with yml files, right?